test-utils/src/test_env.rs

```diff
@@ -235,6 +234,43 @@ pub async fn setup_test_programs_with_accounts(
     )
 }
 
+pub fn init_env_accounts() -> EnvAccounts {
```

Comment:
```suggestion
pub fn get_test_env_accounts() -> EnvAccounts {
```
Naming could be confusing since this method doesn't init anything.
What do you think about this?